src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+tree.addEventListener('click', (e) => {
+  if (e.target.tagName !== 'SPAN') {
+    return;
+  }
+
+  const subTree = e.target.nextElementSibling;
```

Comment:
Consider checking if `e.target.nextElementSibling` is indeed an element before accessing its `tagName` property to avoid potential errors.